src/components/modal/MedicationSaveConfirmModal.tsx

```
@@ -0,0 +1,57 @@
+	onDismiss: () => void;
+	visible: boolean;
+};
+
+export function MedicationSaveConfirmModal({
+	onConfirm,
+	onDismiss,
+	visible,
+}: MedicationSaveConfirmModalProps) {
+	return (
+		<AppModal onClose={onDismiss} variant="center" visible={visible}>
+			<Text style={styles.message}>추가한 약물을 '할일 목록'에 자동으로 저장할까요?</Text>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)